tests/benchmarks/test_coiled.py

```diff
@@ -1,8 +1,14 @@
 import uuid
 
+import pytest
 from coiled import Cluster
 
 
+@pytest.fixture(autouse=True)
+def parquet_benchmark_fixture(benchmark_time):
```

Comment:
As we want to collect timing data for all tests, should we not make this a top-level `autouse` fixture in `conftest.py`?

Reply:
Yes, we can do that -- I've been a little skittish around global `autouse` in case there was some test down the line that we *don't* want to collect benchmarking information on. But I can't think of it right now, so it's a bit theoretical.

Reply:
This is done